style.css

```diff
@@ -338,3 +336,328 @@ input {
+    line-height: 32px;
+    padding: 0;
+    color: #b3bac5;
+  }
+
+  .bar img {
+    width: 18px;
+    height: 21.3px;
+    padding: 5px;
+  }
+
+  .work {
+    padding-left: 10%;
+    padding-right: 10%;
+  }
+
+  .background-card1 {
+    display: flex;
+    background-image: url(imgs/Ellipse19.svg);
+    background-position: left;
+    padding: 50px 10% 88px;
+    width: 80%;
+  }
+
+  .card-img {
+    width: 100%;
+    margin-right: 24px;
+  }
+
+  .card-text {
+    width: 40%;
+    margin-top: -40px;
+  }
+
+  .card-img1 {
+    width: 100%;
+    margin-left: -24px;
+    margin-right: 54px;
+  }
+
+  .card-text h3 {
+    width: 448px;
+    font-family: "Roboto", sans-serif;
+    font-size: 40px;
+    font-weight: 700;
+    line-height: 52px;
+    text-align: left;
+  }
+
+  .card-text p {
+    /* width: 447px; */
+    font-family: "Roboto", sans-serif;
+    font-size: 20px;
+    font-weight: 400;
+    line-height: 28px;
+    text-align: left;
+    margin: 0;
+  }
+
+  .list-items {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    padding: 0;
+    gap: 8px;
+    width: 307px;
+  }
+
+  .background-card2 {
+    display: flex;
+    flex-direction: row-reverse;
+    background-image: url(imgs/Ellipse20.svg);
+    background-position: bottom right;
+    padding: 0 10% 88px;
+    width: 80%;
+  }
+
+  .background-card3 {
+    display: flex;
+    background-image: url(imgs/Ellipse21.svg);
+    background-position: left;
+    padding: 0 10% 0;
+    width: 80%;
+  }
+
+  .background-card4 {
+    display: flex;
+    flex-direction: row-reverse;
+    background-image: url(imgs/Ellipse22.svg);
+    background-position: top right;
+    padding: 88px 10% 0;
+    width: 80%;
+  }
+
+  .work h2 {
+    margin-top: 200px;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 40px;
+    line-height: 24px;
+    display: flex;
+    text-align: center;
+    color: #9390ff;
+    font-family: "Roboto", sans-serif;
+  }
+
+  .work br {
+    display: none;
+  }
+
+  .form-content {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    position: absolute;
+    width: 684px;
+    left: 290px;
+    margin-right: 378px;
+  }
+
+  .form-content p {
+    width: 684px;
+    font-size: 20px;
+    line-height: 28px;
+    padding: 10px 20%;
+    margin-bottom: -40px;
+  }
+
+  .desktop h2 {
+    color: #9093ff;
+    padding: 0;
+    font-size: 32px;
+    text-align: center;
+    margin-left: 20px;
+    font-weight: 500;
+    margin-top: 99px;
+  }
+
+  .row {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    width: 103%;
+  }
+
+  .row #name {
+    margin-left: 6px;
+    margin-right: 6px;
+  }
+
+  .desktop #button {
+    margin-top: 60px;
+  }
+
+  .email-icon,
+  .devider-line hr {
+    position: absolute;
+    width: 1440px;
+    left: 0;
+    top: 4123px;
+    display: flex;
+    justify-content: center;
+    flex-direction: row;
+    margin-left: -100px;
+  }
```

Comment:
- [ ] Also to make the bottom line at the center of the page you should remove the `position`, `width`, and `margin-left`, ie.
```css
.email-icon,
  .devider-line hr {
    position: absolute;
    width: 1440px;
    left: 0;
    top: 4123px;
    display: flex;
    justify-content: center;
    flex-direction: row;
    margin-left: -100px;
  }
```